syncd/FlexCounter.cpp

```diff
@@ -1478,7 +1478,9 @@ void FlexCounter::runPlugins(
     {
         std::to_string(COUNTERS_DB), // TODO
```

Comment:
this needs to co updated to m_dbCounters, and if you need other ASIC_DB and COUNFIG_DB they also needs to be passed to class, since this is hard coding now, and it needs  to be able to support multiple databases